app/svg.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width,initial-scale=1">
+<title>HiGlass</title>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+<link rel="stylesheet" href="hglib.css">
+
+<style type="text/css">
+#demo {
+  position: absolute;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  overflow: hidden;
+  height: 50%;
+}
+#preview {
+  position: absolute;
+  top: 50%;
+}
+</style>
+
+<script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+<script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.8.2/pixi.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/react-bootstrap/0.32.1/react-bootstrap.min.js"></script>
+<script src="http://d3js.org/d3.v5.min.js"></script>
+</head>
+<body>
+  <div id="demo"></div>
+  <div id="preview"></div>
+</body>
+<script src='hglib.js'></script>
+<script>
+d3.json(location.search.slice(1).replace('http://higlass.io/app/?config=', 'http://higlass.io/api/v1/viewconfs/?d=') || 'viewConf.json').then((viewConfig) => {
```

Comment:
Sorry, what is the `location.search.slice(1).replace` statement doing?

Also, just out of curiosity, why `d3.json` over `fetch`?

Reply:
> Sorry, what is the location.search.slice(1).replace statement doing

The idea is that you can do this:
```
http://localhost:8080/svg.html?http://higlass.io/app/?config=MSHhOBbOSW6iIovB5yk6BA
```
and it will translate that into the url for the json, and view that. (Or you can reference a viewconf you've saved locally, or you can leave the query blank, and get the default.)

Maybe a little input box on the page that would prompt for a url to make this functionality more obvious?

> Also, just out of curiosity, why d3.json over fetch?

You mean the built in Fetch API? No good reason. I'll switch that over.